app/models/order.js

```diff
@@ -61,5 +61,10 @@ export default Model.extend(LocationHashable, {
   totalPrice(orderItems, shipping) {
     const orderItemsTotal = orderItems.reduce((acc, cur) => acc + Number(cur.get("quantity") * cur.get("unitPrice")), 0);
     return orderItemsTotal + shipping;
+  },
+
+  @computed("orderItems.[]")
+  isValid(orderItems) {
+    return orderItems.get("length") !== 0;
```

Comment:
Looks good. I forgot to mention that we can use the `gt` helper to make this more compact: http://emberjs.com/api/classes/Ember.computed.html#method_gt

```js
isValid: gt("orderItems.length", 0)
```

Or you can also use the notEmpty: http://emberjs.com/api/classes/Ember.computed.html#method_notEmpty

```js
isValid: notEmpty("orderItems")
```